components/automate-ui/src/app/entities/client-runs/client-runs.effects.ts

```diff
@@ -38,11 +36,6 @@ export class ClientRunsEffects {
       ofType(ClientRunsActionTypes.GET_NODES),
       withLatestFrom(this.store),
       switchMap(([_action, storeState]) => {
-        const sidebarState: SidebarState = storeState.sidebar;
-        const clientRunsState: ClientRunsEntityState = storeState.clientRunsEntity;
-        const nodeFilter = clientRunsState.nodeFilter;
-        nodeFilter.organizations = sidebarState.selectedOrgs;
-        nodeFilter.servers = sidebarState.selectedChefServers;
```

Comment:
Removing selection to the chef server and orgs sidebar from client runs get nodes request.